ansible/install-postgresql.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: localhost
+  gather_facts: no
+  vars:
+    instance_name: ooni-measurements-db
+    disk_size: 200
+    ram_size: 4096
+    cpu_cores: 2
+  roles:
+    - { role: eclipsis, when: api_token is defined }
```

Comment:
It makes sense though eclipsis role is not a heavy dependency to run this role; one can still use to setup the postgresql role.